packages/notification-center/src/store/notification-center.context.ts

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
+
 import { IMessage, ButtonTypeEnum } from '@novu/shared';
+import { IUserPreferenceSettings } from '@novu/client';
+
 import { INotificationCenterContext, ITab } from '../shared/interfaces';
 
 export const NotificationCenterContext = React.createContext<INotificationCenterContext>({
   onUrlChange: (url: string) => {},
   onNotificationClick: (notification: IMessage) => {},
   onUnseenCountChanged: (unseenCount: number) => {},
   onActionClick: (identifier: string, type: ButtonTypeEnum, message: IMessage) => {},
+  onTabClick: (tab: ITab) => {},
+  preferenceFilter: (userPreference: IUserPreferenceSettings) => {},
```

Comment:
I think you need to add it to component web, vue and angular components

Reply:
thanks for the comment i thought it was autogenerated, i will look into it.